doc/integrator/filter.rst

```diff
@@ -66,6 +66,17 @@ Simple example:
             ...
         },
 
+Configuration - Administration interface step
```

Comment:
same, `Administration interface` ...

Reply:
IIUC what is important here is that two steps are needed to get this feature to work,
one in the project file, one in the administration interface. 

So maybe to make it even clearer we could have a header "Configuration", and within this paragraph, something like "Step 1: project file", "Step 2: administration interface".

Reply:
IIUC?

Reply:
If you think that important I prefer your sentence than current one... or just "#1 project file", "#2 administration interface"?